### ISSUE TITLE
Does pnpm support something like lerna version ?

### ISSUE THREAD

**Issue by lake2** [pnpm/pnpm#5762]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

**Describe the user story**

A clear and concise description of what workflow is meant to be improved.
Example: "As a developer, I often want to do <something>, but I often face <problem>".

**Describe the solution you'd like**

A clear and concise description of what you want to happen. Consider that pnpm is used
by many people, and your particular use case might not make sense to implement in the core.

**Describe the drawbacks of your solution**

This section is important not only to identify future issues, but also for us to see whether
you thought through your request. When filling it, ask yourself what are the problems we could
have maintaining what you propose. How often will it break?

**Describe alternatives you've considered**

A clear and concise description of any alternative solutions or features you've considered,
and why you think they wouldn't be good enough.

**Comment by zkochan** [comment]
No, but Lerna works with pnpm. So you can use Lerna for versioning.
Or [changesets](https://pnpm.io/using-changesets).

**Comment by zkochan** [comment]
This is implemented in current pnpm, so closing.

pnpm has changesets built in: `pnpm change` records a change intent with the bump type and changelog text, and `pnpm version -r` applies the pending intents across the workspace, with `pnpm publish -r` to publish.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
